Updating PR template & adding exclusion tags

wallentx opened the pull request:
<!-- Merging Requirements:
- Please give your PR a title that is release-note friendly
- In order to be merged, you must add the most appropriate category Label (Added, Changed, Fixed) to your PR
- Use the prompts below to provide information wherever applicable
-->
<!-- What is the purpose of the changes in this PR? -->
Purpose: PR to source input for changes to the PR template


<!-- What is the current behavior?** (You can also link to an open issue here) -->
Current Behavior: The PR template is pretty basic, and there have been some requests to modify some aspects of it.


<!-- What is the new behavior (if this is a feature change)? -->
New Behavior: TBD


<!-- Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?) -->
Breaking Changes: None


<!-- Testing notes (is this code covered by tests, or equivalent manual testing?) -->
Testing Notes: none


<!-- Are there any visual examples to help explain this PR? (attach any .gif/movie/console output below) -->

wallentx committed:
Updating PR template & adding exclusion tags

arvidn commented:
I think the comments become a distraction to filling in this form. Also, most of my PRs don't fit this form well. The main encouragement I'd like to see in this template are something like:

* explain why this is an improvement. Does it add missing functionality, improve performance or reduce complexity?
* attach any supporting evidence

Maybe there should be a check-box saying "I have ensured there is unit test coverage of this change, both success cases, failure cases and edge cases"

wallentx committed:
Updating PR Template, and additional release note filtering label

wallentx commented:
Adjusted template:
```
<!-- Merging Requirements:
- Please give your PR a title that is release-note friendly
- In order to be merged, you must add the most appropriate category Label (Added, Changed, Fixed) to your PR
-->
<!-- Explain why this is an improvement (Does this add missing functionality, improve performance, or reduce complexity?) -->
Purpose:


<!-- Does this PR introduce a breaking change? -->
Current Behavior:
New Behavior:


<!-- As we aim for complete code coverage, please include details regarding unit, and regression tests -->
Testing Notes:


<!-- Attach any visual examples, or supporting evidence (attach any .gif/video/console output below) -->
```
